src/resize.c

```diff
@@ -371,7 +382,7 @@ void ResizeClientKeyboard(ClientNode *np, MouseContextType context)
          DiscardMotionEvents(&event, np->window);
 
          UpdateSize(np, context, event.xmotion.x, event.xmotion.y,
-                    startx, starty, oldx, oldy, oldw, oldh);
+                    startx, starty, oldx, oldy, oldw, oldh, 0);
```

Comment:
It seems to me that either `UpdateSize` needs to check for the case that prevContext is NULL or this should pass a valid pointer into UpdateSize?

Reply:
You mean passing something else instead of 0?

Reply:
`0` turns into a null pointer dereference in UpdateSize, which leads to a seg fault for me when I select `resize` from the window menu.  I think passing `&context` might give the desired effect.

Reply:
I see. As I said, I'm definitely not an expert at C so feel free to modify the code as you see fit.